bciers/apps/reporting/src/app/components/products/ProductionDataForm.tsx

```diff
@@ -50,6 +54,17 @@ const ProductionDataForm: React.FC<Props> = ({
   };
 
   const onSubmit = async (data: any) => {
+    /*
+      Handle pulp & paper overlapping industrial process exception:
+      If pulp & paper is reported and there are industrial process emissions that are also categorized as excluded (ie: woody biomass)
+      Then the 'Pulp and paper: chemical pulp' product must be reported
+    */
+    if (isPulpAndPaper && overlappingIndustrialProcessEmissions > 0) {
+      if (!data.product_selection.includes("Pulp and paper: chemical pulp")) {
+        setErrors(["Missing Product: 'Pulp and paper: chemical pulp'"]);
```

Comment:
Should we add to the message: "Please add the product on the operation or facility information page" ?